### PR TITLE
Implement IngredientService (#11) and Fix Issue #33 

### PR DESCRIPTION
**Pull request by JoyfulReaper**
Implement IngredientService (#11) and Fix Issue #33

### CODE REVIEW

File: CRUDRecipeEF.BL.DL/Services/IngredientService.cs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        }
+
+
+        /// <summary>
+        /// Gets an ingredient by name
+        /// </summary>
+        /// <param name="name"></param>
+        /// <returns>Ingredient</returns>
+        /// <exception cref="KeyNotFoundException"></exception>
+        public async Task<IngredientDetailDTO> GetIngredientByName(string name)
```
Comment (chrisK00) [inline CRUDRecipeEF.BL.DL/Services/IngredientService.cs:99]: i suggest we remove this considering we would never need to get an ingredient by its name

**Review by chrisK00 — APPROVED** [review]
Looks good

File: CRUDRecipeEF.BL.DL/Services/IngredientService.cs (hunk 1 above)
Comment (JoyfulReaper) [reply CRUDRecipeEF.BL.DL/Services/IngredientService.cs:99]: This is currently used in both the IngredientMenu (line 94) and RecipceMenu (line 149).

Comment (chrisK00) [reply CRUDRecipeEF.BL.DL/Services/IngredientService.cs:99]: ahh right, makes sense to keep it then.

**Pull request merged**